http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -966,7 +978,7 @@ class GKEInterpreter[F[_]](
         .namespaceExists(dbApp.cluster.getClusterId, KubernetesNamespace(dbApp.app.appResources.namespace))
         .map(!_) // mapping to inverse because booleanDoneCheckable defines `Done` when it becomes `true`...In this case, the namespace will exists for a while, and eventually becomes non-existent
 
-      _ <- streamUntilDoneOrTimeout(fa, 30, 5 seconds, "delete namespace timed out")
+      _ <- streamUntilDoneOrTimeout(fa, 60, 5 seconds, "delete namespace timed out")
```

Comment:
Why the increase in timeout here?

Reply:
I increased this initially becuz I thought deleting namespace was stuck, and increasing timeout is all we needed. In the end, I realize in some cases, namespaces are really just stuck in deleting, increasing timeout wouldn't help.

But regardless, I think increasing the timeout is nice to do anyway and there's not much downside in this case.